runtime/doc/lua.txt

```diff
@@ -242,10 +242,12 @@ The Lua print() function redirects its output to the Nvim message area, with
 arguments separated by " " (space) instead of "\t" (tab).
 
                                                                   *:lua=* *:lua*
-:lua {chunk}
-    Executes Lua chunk {chunk}. If {chunk} starts with "=" the rest of the
-    chunk is evaluated as an expression and printed. `:lua =expr` or `:=expr` is
-    equivalent to `:lua print(vim.inspect(expr))`.
+:[range]lua {chunk}
```

Comment:
If `[range]` and `{chunk}` are mutually exclusive, I would expect `:[range]lua` and `:lua {chunk}` to each be documented in separate paragraphs (c.f., `:help :read`).